File: day1/cards/public/grader.cpp
#include "cards.h"
#include <iostream>

int main() {
    int T;
    std::cin >> T;
    for (int i = 0; i < T; i++) {
        int X, Y, Z;
        std::cin >> X >> Y >> Z;
        std::cout << maximum_score(X, Y, Z) << '\n';
    }
    return 0;
}

File: day1/cards/public/grader.py
import cards
import sys

T = int(input() )

for _ in range(T):
    a_str, b_str, c_str = input().split()
    a = int(a_str)
    b = int(b_str)
    c = int(c_str)

    result = cards.maximum_score(a, b, c)

    print(result)